<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCSAddDlg</class>
 <widget class="QDialog" name="BCSAddDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>38</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#horizontalWidget{
	background-color:#505050;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>28</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>306</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>306</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>56</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>280</width>
             <height>56</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(250, 250, 250);
</string>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>66</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>条码秤编号:</string>
            </property>
           </widget>
           <widget class="QLabel" name="bcsCodeLabel">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>20</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>001</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>250</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>280</width>
             <height>250</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_21">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>关联商品组：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_17">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>30</y>
              <width>62</width>
              <height>30</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>类型：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_19">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>70</y>
              <width>52</width>
              <height>30</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>IP：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="IPLineEdit">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>70</y>
              <width>146</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>146</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>146</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-width:1px;
border-color: #ed9a6e;
border-style:solid;
background-color: rgb(246, 246, 246);
</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushFrom">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>180</y>
              <width>243</width>
              <height>42</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>243</width>
              <height>42</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>243</width>
              <height>42</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#pushFrom{
 	color:#FFFFFF;
	font-size:16px;
	border-image: url(:/shareImages/sure_normal.png);
}
QPushButton#pushFrom:pressed {
	border-image: url(:/shareImages/sure_press.png);
}</string>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
           <widget class="QComboBox" name="typeComboBox">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>30</y>
              <width>146</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>146</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>146</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:black;
background-color: rgb(246, 246, 246);
QComboBox QAbstractItemView::item{
	min-height:30px;
	height:30px;
	background-color: rgb(246, 246, 246);
};
QComboBox{
	border-width:1px;
	border-color: #ed9a6e;
	border-style:solid;
	background-color: rgb(246, 246, 246);
}
</string>
            </property>
            <item>
             <property name="text">
              <string>RL00</string>
             </property>
            </item>
           </widget>
           <widget class="QComboBox" name="goodsGroupComboBox">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>110</y>
              <width>146</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>146</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>146</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:black;
background-color: rgb(246, 246, 246);
QComboBox QAbstractItemView::item{
	min-height:30px;
	height:30px;
	background-color: rgb(246, 246, 246);
};
QComboBox{
	border-width:1px;
	border-color: #ed9a6e;
	border-style:solid;
	background-color: rgb(246, 246, 246);
}
</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="WndDigitalKeyboard" name="DigitalKeyWnd" native="true">
        <property name="minimumSize">
         <size>
          <width>303</width>
          <height>303</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WndDigitalKeyboard</class>
   <extends>QWidget</extends>
   <header>WndDigitalKeyboard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
